src/main/proto/commit.proto

```diff
@@ -24,5 +24,7 @@ message Commit {
     string author_email = 4;
     string date = 5;
     bool qommit = 6;  // Quality commit.
-    repeated CommitStats commit_stats = 7;  // Stats on a commit.
+    int32 num_lines_add = 7;  // Number of lines for whole commit.
```

Comment:
Makes sense. In such a case, it makes sense to make their purpose obvious by provide a detailed comment for this field in the proto file.